src/js/BookReader.js

```diff
@@ -302,12 +302,58 @@ BookReader.prototype.initParams = function() {
     }
   }
 
+  // Check for Search plugin
+  if (this.options.enableSearch) {
+    // If initialSearchTerm not set
+    if (!this.options.initialSearchTerm) {
+      // Look for any term in URL
+      if (params.search) {
+        // Old style: /search/[term]
```

Comment:
Q: at this point - will we even have `/search/term` in URL?

Reply:
will `params` be out of sync?

Reply:
>Q: at this point - will we even have /search/term in URL?
in the hash URL styles, yes
> will params be out of sync in any way?
No, not resetting